tests/src/system/basic/WskBasicTests.scala

```diff
@@ -304,64 +251,76 @@ class WskBasicTests
      * surely when it runs there should be some indication in the logs. Don't
      * think this is true currently.
      */
-    it should "create and invoke action with malformed js resulting in activation error" in {
-        val name = "MALFORMED"
-        wsk.action.sanitize(name)
-        createAction(name, TestUtils.getTestActionFilename("malformed.js"))
-
-        val activation = wsk.action.invoke(name, Map("payload" -> "whatever".toJson))
-        val activationId = wsk.action.extractActivationId(activation)
-        activationId shouldBe a[Some[_]]
-
-        val expected = "ReferenceError" // representing nodejs giving an error when given malformed.js
-        val (found, logs) = wsk.activation.contains(activationId.get, expected)
-        assert(found, s"Did not find '$expected' in activation($activationId) ${logs getOrElse "empty"}")
+    it should "create and invoke action with malformed js resulting in activation error" in withAssetCleaner(wskprops) {
+        (wp, assetHelper) =>
+            val name = "MALFORMED"
+            assetHelper.withCleaner(wsk.action, name) {
```

Comment:
To me this looks like the action dies at the end of the scope, but it really dies at the end of the `withAssetCleaner` scope?